src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -389,3 +477,33 @@ export const AccessibleViewAction = registerCommand(new MultiCommand({
 	}],
 }));
 
+
+class AccessibleViewSymbolQuickPick {
+	constructor(private _accessibleView: AccessibleView, @IQuickInputService private readonly _quickInputService: IQuickInputService) {
+
+	}
+	show(provider: IAccessibleContentProvider): void {
+		const quickPick = this._quickInputService.createQuickPick<IAccessibleViewSymbol>();
+		const picks = [];
+		const symbols = this._accessibleView.getSymbols();
+		if (!symbols) {
+			return;
+		}
+		for (const symbol of symbols) {
+			picks.push({
+				label: symbol.label,
+				ariaLabel: symbol.ariaLabel
+			});
+		}
+		quickPick.canSelectMany = false;
+		quickPick.items = symbols;
+		quickPick.show();
+		quickPick.onDidAccept(() => {
+			this._accessibleView.showSymbol(provider, quickPick.selectedItems[0]);
```

Comment:
This isn't going to dismiss the quick pick. Is that expected?

Reply:
handled #189659  😄

Reply:
accepting one does because of the focus change. escaping the quick pick wasn't handled, but now is #189658

Reply:
There's a setting called ignoreFocusOut that won't close the quick pick when focus is lost.

As a result this qp won't go away with that setting. I recommend explicitly closing it